earth_enterprise/src/fusion/autoingest/tools/genewmapdatabase.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Google Inc.
+// Copyright 2018 Google Inc.
```

Comment:
This is the wrong way to update copyright notices.  You should leave the year of first publication.  You can optionally, include a year range.  Here's an explanation, even if it may not be the best source: https://www.copyrightlaws.com/copyright-notice-year/

Since this is not being developed by Google any more, you can leave the first notice and add another one, so it looks something like:

```
// Copyright 2017 Google Inc.
// Copyright 2018 the Open GEE Contributors
```

Or, I think, all parties are OK with switching over to only a notice mentioning the Open GEE contributors for files modified after the Google release.

Reply:
I updated the changes to the copyright. I reverted the Google copyrights to their previous state and added and OpenGEE contributor copyright where applicable. Thanks for the feedback!

Reply:
:+1: Looks good to me.